# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/CacheConstants.h
# Commit: 6591f14bbc71
# Full Hash: 6591f14bbc713f11b6316712d931ecc19a443a5c
# Author: Morgan Reschenberg <mreschenberg@mozilla.com>
# Date: 2021-10-05 03:52:32
# Regressor Bug: 1726227
# File Overlap Count: 1
# Description:
#   Bug 1726227: Cache parent-relative accessible bounds, resolution in parent process r=Jamie,eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123399
# ==============================================================================

diff -r 06dda9072cb7 -r 6591f14bbc71 accessible/base/CacheConstants.h
--- a/accessible/base/CacheConstants.h	Mon Oct 04 19:28:49 2021 +0000
+++ b/accessible/base/CacheConstants.h	Mon Oct 04 19:46:54 2021 +0000
@@ -15,6 +15,7 @@
   static constexpr uint64_t NameAndDescription = ((uint64_t)0x1) << 0;
   static constexpr uint64_t Value = ((uint64_t)0x1) << 1;
   static constexpr uint64_t Bounds = ((uint64_t)0x1) << 2;
+  static constexpr uint64_t Resolution = ((uint64_t)0x1) << 3;
   static constexpr uint64_t All = ~((uint64_t)0x0);
 };
 